### ISSUE TITLE
Image resize (v2.2.0) not resizing to consistent or specified size

### ISSUE THREAD

**Issue by farflungideas** [moxiecode/plupload#1425]
Tried to implement the new higher quality image resizing (which works really well from an image quality perspective), however, if you take a large image and set the height / width parameters, it doesn't scale the image evenly (like the old version did).

eg. I upload an image of size: 5054 x 3370

Then set plupload up as such:

```javascript
resize: { 
  width: 1008,
  height: 1512,
  quality: 95,
  preserve_headers: false
},
```

plupload resizes the file to: 1511 x 1007... Which is not expected.

This was initially mentioned here: https://github.com/moxiecode/plupload/issues/707#issuecomment-260711980
And a new ticket requested here: https://github.com/moxiecode/plupload/issues/707#issuecomment-260870219

Thanks again!

**Comment by farflungideas** [comment]
@jayarjo - can you confirm the existence of this bug, and that it is in fact a bug?

**Comment by jayarjo** [comment]
Has been addressed in 2.3.3.
